<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mon Editeur de Texte</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutZoneTexte" stretch="1,3">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEditContenu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>15</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutBas" stretch="0,0">
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>90</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QComboBox" name="comboBoxCouleur">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutTexteStyle">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QCheckBox" name="checkBoxGras">
           <property name="toolTipDuration">
            <number>10</number>
           </property>
           <property name="text">
            <string>Gras</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QCheckBox" name="checkBoxItalique">
           <property name="text">
            <string>Italique</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxPolice">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Police</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioBtnCourier">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Courier</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtnArial">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Arial</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtnTNR">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Times New Roman</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer_Param_tres"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
  </action>
  <action name="actionEnregistrer_Param_tres">
   <property name="text">
    <string>Enregistrer Param√®tres</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
